lib/src/api/headers/mod.rs

```diff
@@ -0,0 +1,14 @@
+//! HTTP headers used by SurrealDB
+
+use reqwest::header::HeaderName;
+
+pub static ID: HeaderName = HeaderName::from_static("sur-id");
```

Comment:
I don't know how others feel about this but I would prefer we use `surrealdb` in full instead of `sur`.